### PR TITLE
[15.0][ADD] website_sale_stock_provisioning_time.

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
There's already https://github.com/OCA/e-commerce/tree/13.0/website_sale_stock_provisioning_date. Migrate it instead.

**Comment by jans23** [comment]
Both modules work a bit different. With module website_sale_stock_provisioning_date we couldn't display the expected provisioning time (in work days) on a particular product and it requires the expected stock move to be planned in Odoo. In our workflow we don't have such expected stock move but for some products we require several days to prepare an individual order before it can be shipped. We didn't find website_sale_stock_provisioning_date suitable to reflect such.

**Comment by pedrobaeza** [comment]
OK, please reflect such differences in the README.

File: website_sale_stock_provisioning_time/models/product_template.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    provisioning_time_in_workdays = fields.Integer(
+        string="Provisioning time in workdays"
```
Comment (manengel) [inline website_sale_stock_provisioning_time/models/product_template.py]: workdays --> work days I think.

Comment (jans23) [reply website_sale_stock_provisioning_time/models/product_template.py]: It seems both to be ok:

* https://dict.leo.org/englisch-deutsch/arbeitstage
* https://www.dict.cc/?s=arbeitstage
* https://www.deepl.com/translator#de/en/arbeitstage

Comment (manengel) [reply website_sale_stock_provisioning_time/models/product_template.py]: O.K. then the module does look good to me.

**Comment by pedrobaeza** [comment]
This pull request is incorrect containing a lot of undesired changes. Closing, and if re-submitted, it should be with another name that don't collapse with the existing one.

**Pull request closed without merging**